components/atoms/InsightBadge/insight-badge.tsx

```diff
@@ -0,0 +1,17 @@
+import Text from "components/atoms/Typography/text";;
+
+interface BadgeProps {
+    isPublic: boolean
+}
+
+const Badge = ({ isPublic }: BadgeProps): JSX.Element => {
+  return (
+    <div className="inline-flex border border-light-slate-6 rounded-[26px] bg-light-slate-3 px-2.5 py-1">
```

Comment:
Please avoid using rounded-[26px] this is an anti-pattern in **open-sauced**. In tailwind, the builtin  max class is `.rounded-3xl` and in px is `16px`

Reply:
I agree with this. This eventually doesn't scale.

Reply:
Since tailwind does not have a builtin class that matches the value given on the design (26px) is it acceptable to create a global style and apply there or use the built in max class which is also visually acceptable?

Reply:
Yes, that is fine. A global class makes it scale if we need to change in multiple places. 

please name it `rounded-badge`

Reply:
disagree with global classes, use rem if this isn't design-system related (it's not), the problem is not using responsive values.